superdesk/storage/fix_href.py

```diff
@@ -0,0 +1,95 @@
+import copy
+import superdesk
+
+from flask import current_app as app
+from superdesk.editor_utils import generate_fields, get_field_content_state
+
+
+class FixHrefCommand(superdesk.Command):
+    """Fix href links in db."""
```

Comment:
It would be beneficial if we had a more descriptive docstring here including examples, like we have with other commands, so it is easier to understand from the `readthedocs` documentation.